---
title: "Computational Biology"
layout: single-portfolio
excerpt: "<img src='/images/research/comp-bio.png' alt=''>"
collection: research
order_number: 20
header: 
  og_image: "research/Agreement-Strength.png"
---

Modern breakthroughs in assays and computing have enabled the massive collection of data on complex biological systems. I develop computational biology methods to improve our understanding of gene regulation and better inform *in vitro* scientific experiments. I have worked on projects including combinatorial biomarker identification, drug repurposing, experimental design, cellular aging, and nuclear mechanogenomics. 

## Articles

**Cammarata, L.**, Uhler, C. and Shivashankar, G.V., 2023. Adhesome Receptor Clustering is Accompanied by the Co-localization of the Associated Genes in the Cell Nucleus. _bioRxiv_.
> In this project, we characterized map between protein assemblies on the cell membrane and 3D organization of the genome in nucleus, specifically for adhesome in IMR-90 fibroblasts using multiple genomic/epigenomic data modalities (RNA-seq, ChIP-seq, Hi-C, FISH).

[Article](https://www.biorxiv.org/content/10.1101/2023.12.07.570697v1){: .btn--research} [GitHub Repo](https://github.com/uhlerlab/adhesome_clustering/){: .btn--research}

Braunger\*, J.M., **Cammarata\*, L.V.**, Sornapudi, T., Uhler, C. and Shivashankar, G.V., 2023. Transcriptional changes are tightly coupled to chromatin reorganization during cellular aging. _Aging Cell_.
> In this project, we developed novel a time-course prize-collecting Steiner tree algorithm to identify age-associated transcription factors and investigated the coupling between gene transcription and spatial gene clustering in aging (using RNA-seq, ChIP-seq, Hi-C). Our work got media coverage in the [Eric and Wendy Schmidt Center](https://www.ericandwendyschmidtcenter.org/updates/researchers-identify-new-regulators-of-cellular-aging).

[Article](https://onlinelibrary.wiley.com/doi/full/10.1111/acel.14056){: .btn--research} [GitHub Repo]([https://github.com/uhlerlab/adhesome_clustering/](https://github.com/uhlerlab/aging_transcriptome_and_chromatin)){: .btn--research}

Zhang, J., **Cammarata\*, L.V., Squires\*, C.**, Sapsis, T.P. and Uhler, C., 2023. Active learning for optimal intervention design in causal models. _Nature Machine Intelligence_.
> In this work, we developed a consistent causal active learning strategy to identify optimal interventions in linear causal models with known causal graph for the discrepancy between the post-interventional mean of the distribution and a desired target mean.

[Article](https://www.nature.com/articles/s42256-023-00719-0){: .btn--research} [GitHub Repo](https://github.com/uhlerlab/actlearn_optint){: .btn--research}

Belyaeva\*, A., **Cammarata\*, L.**, Radhakrishnan\*, A., Squires, C., Yang, K.D., Shivashankar, G.V. and Uhler, C., 2021. Causal network models of SARS-CoV-2 expression and aging to identify candidates for drug repurposing. _Nature Communications_.

> In this project, we developed a drug repurposing pipeline using overparametrized representation learning, network analysis and causal structure learning and applied it to identify *in silico* candidate drug targets for the treatment of COVID-19. Our work got media coverage in the [MIT News Office](https://news.mit.edu/2021/machine-learning-treatment-covid-19-0216) and other specialized online media.

[Article](https://www.nature.com/articles/s41467-021-21056-z){: .btn--research} [GitHub Repo](https://github.com/uhlerlab/covid19_repurposing){: .btn--research}

Delaney\*, C., Schnell\*, A., **Cammarata\*, L.**, Yao-Smith, A., Regev, A., Kuchroo, V.K. and Singer, M., 2019. COMET: a tool for marker-panel selection from single-cell transcriptomic data. _Molecular systems biology_.

> In this project, we developed COMET, a computational tool for combinatorial marker detection from single-cell transcriptomic data available as an online platform and a Python package and leveraged COMET to identify B cell subpopulations.

[Article](https://www.embopress.org/doi/full/10.15252/msb.20199005){: .btn--research} [COMET online platform](http://www.cometsc.com/index){: .btn--research} [Python package](https://hgmd.readthedocs.io/en/latest/){: .btn--research} [GitHub Repo](https://github.com/MSingerLab/COMETSC){: .btn--research}

## Manuscripts in preparation

